README.md

```diff
@@ -0,0 +1,58 @@
+# EdgeCast CLI
+
+This is the official command line interface (CLI) for EdgeCast.
+
+## Getting Started
+
+### Requirements
+- Go 1.17
+
+### Installation
+To install the CLI, use the makefile:
+```
+$ make install
+```
+
+### Using the CLI
+Commands follow the following format:
+```
+$ ec <command> <subcommand> --arg1=value1 --arg2=value2`
+```
+
+Below is an example:
+```
+$ ec origin getallorigins --accountnumber=ABCDE --mediatypeid=3
+```
+
+In addition to using flags to provide inputs, you may also provide JSON input:
+```
+$ ec origin getallorigins --input='{AccountNumber:ABCD, MediaTypeID=3}'
+```
+
+To view the top-level documentation for the CLI, use the `help` command:
```

Comment:
Including that this is also how to view all fo the commands will add clarity.

> To view the top-level documentation and available commands for the CLI, use the `help` command:

Reply:
Good suggestion @defstream updated